site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -1,201 +1,216 @@
-import { skipOn } from '@cypress/skip-test';
-skipOn(Cypress.env('run_area') === 'CI', () => {
-    describe('Test Rainbow Grading', () => {
-        beforeEach(() => {
-            cy.login('instructor');
-            cy.visit(['sample', 'config']);
-            checkCheckbox('[data-testid="display-rainbow-grades-summary"]');
-            cy.visit(['sample', 'reports', 'rainbow_grades_customization']);
-            reset();
+import { buildUrl } from '/cypress/support/utils.js';
+
+describe('Test Rainbow Grading', () => {
+    beforeEach(() => {
+        // instructor2 has access to the testing course
+        cy.login('instructor2');
+        cy.visit(['testing', 'config']);
+        cy.get('[data-testid="display-rainbow-grades-summary"]').check();
+        cy.visit(['testing', 'reports', 'rainbow_grades_customization']);
+        cy.get('[data-testid="display-grade-summary"]').should('be.visible'); // Ensure page is loaded
+        reset();
+    });
+    it('Add grades to numeric gradeable', () => {
+        const gradesfile = 'cypress/fixtures/rainbowgrades_ci_numeric.csv';
+
+        cy.visit(['testing', 'gradeable', 'numeric', 'grading']);
+        cy.get('#csvUpload').selectFile(gradesfile, { action: 'drag-drop' });
+        cy.on('window:confirm', () => true);
+
+        cy.visit(['testing', 'gradeable', 'numeric', 'quick_link?action=open_grading_now']);
+        cy.visit(['testing', 'gradeable', 'numeric', 'quick_link?action=release_grades_now']);
```

Comment:
Ideally this would probably be a part of a different it() block, or in a before() block, but if it would be too difficult, its okay.